src/server/qgsserverresponse.h

```diff
@@ -157,6 +157,24 @@ class SERVER_EXPORT QgsServerResponse
      * Reset all headers and content for this response
      */
     virtual void clear() = 0;
+
+    /**
+     * Get the data written so far
+     *
+     * This is implementation dependant: some implementations may not
+     * give access to the underlyng and return an empty array.
```

Comment:
s/underlyng/underlying/ 

BTW  don't get why you stress that it might return an empty array. Are you thinking at the a streaming response case?
In that case, the reason for getting an empty array would be dependant on the moment that this function is called: if it's called in `responseComplete()` you might get an empty array, but if called in `sendResponse()` you should get the current chunk, as you correctly mention here below.

Or am I missing something?